### ISSUE TITLE
[BUG] Inventory Tweaks Renewed blocking Tinkers Crafting Station GUI

### ISSUE THREAD

**Issue by JTurner-Portfolio** [YahiaTGF/MC-Odyssey-2#4]
**Describe the bug**
When placing a Tinker's crafting station beside an inventory and then accessing the crafting station, inventory tweaks sort buttons block a portion of the crafting GUI

**To Reproduce**
1. Place container of some sort e.g. a vanilla chest
2. Place a Tinker's Construct crafting station adjacent to the container
3. Access crafting table and view GUI

**Expected behavior**
Inventory tweaks GUI to wrap around Tinkers GUI

**Screenshots**
![MCO2 InvTweaks](https://user-images.githubusercontent.com/94337000/141717952-9bfc4cc3-af77-49ff-aa9b-5184c6c95e6a.PNG)

**Information (please complete the following information):**
 - Windows 10
 - Pre-release 01
 - 16 GB
 - Java 1.8.0.301

**Additional context**
I'm not 100% sure that the problem stems from inventory tweaks or rather some other mod, I did find a reddit post with the same issue being linked to inventory tweaks though. The solution in that thread didn't work for me so I can't confirm.

**Comment by JTurner-Portfolio** [comment]
I found that the rightmost sort button and the one above and to the left of that is indeed inventory tweaks renewed. I removed those by turning off the buttons in the config. The six smaller buttons are from quark and 5 of the 6 are easily removable by disabling management in quark's config. The sixth button is the filter button at the far left of the screenshot, it seems like the only way to remove that is to blacklist each individual GUI that it interferes with.
